### PR TITLE
Feat/181 martech

### LINKED ISSUE #181: Integrate martech (delayed approach)
Tasks to be done:
1. add the OneTrust script/code snippet in `delayed.js`
2. add the stage/prod (depending if localhost/.hlx. vs prod domain) Adobe Launch script in `delayed.js` (as a temporary solution for quickly integrating the martech while keeping LHS green)
3. configure CSP headers (https://www.hlx.live/docs/custom-headers) for all domains called from the launch script (e.g. `assets.adobedtm.com`), which will be failing otherwise with CORS issues
4. in `delayed.js`, before loading the Launch script, init the datalayer
5. in `delayed.js`, after loading the Launch script, push page load event to the datalayer with the payload as specified in the SDR

[Real Madrid Project -> Site Analytics Instrumentation](https://wiki.corp.adobe.com/display/AEMSites/ECSA+-+Analytics+Omnivore+-+Real+Madrid+Project#ECSAAnalyticsOmnivoreRealMadridProject-SiteAnalyticsInstrumentation) -> the docs attached under `Real Madrid new SDR and implementation plan (in Spanish)` have enough detail on how to do this, just that instead of loading in `head` as the agency says, we do it in `delayed.js` and check the LHS to be still ~100.

Use the [AEP Debugger Chrome extension](https://experienceleague.adobe.com/docs/platform-learn/data-collection/debugger/overview.html?lang=en) to validate data is collected properly via the Launch script.

### PR DESCRIPTION
**Pull request by chicharr**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #181

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu/colegios/combinado-parque-warner
- After: https://feat-181-martech--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu/colegios/combinado-parque-warner

### CODE REVIEW

File: sites/scripts/delayed.js [hunk 1]
```diff
@@ -1,7 +1,92 @@
...
+}
+// End one trust
+
+// Init Adobe data layer
+window.adobeDataLayer = window.adobeDataLayer || [];
+window.adobeDataLayerInPage = true;
+
+// Load Adobe Experience platform data collection (Launch) script
+if (!window.location.host.includes('hlx.page') && !window.location.host.includes('localhost')) {
+  loadScript('https://assets.adobedtm.com/ab05854e772b/7bc47c0b7114/launch-d2e30cc4a650.min.js');
```
[the remark this answers is not in the document]
Comment (chicharr) [reply sites/scripts/delayed.js:88]: to be consistent with the rest of the setup, I will add them to the placeholders.
However, I'm thinking if it is a good idea to have all these technical settings editable by authors.

File: sites/scripts/scripts.js [hunk 2]
```diff
@@ -489,6 +489,25 @@ export function getNavLink(link, external = true) {
   return `${pathname}${searchParams}`;
 }
 
+/**
+ * Loads the script in the head section
+ * @param {string} url and other attrs
+ * @returns script
+ */
+export function loadScript(url, attrs) {
+  const head = document.querySelector('head');
```
[the remark this answers is not in the document]
Comment (chicharr) [reply sites/scripts/scripts.js:498]: is there a reason for that? I got this method from [other franklin projects ](https://github.com/search?q=org%3Ahlxsites%20loadScript&type=code)

File: sites/scripts/delayed.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (iuliag) [reply sites/scripts/delayed.js]: That's a good debate with no definite answer. For customers that have marketers/Analytics dedicated teams, they'd want to be able to replace those without involving development.

File: sites/scripts/scripts.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (iuliag) [reply sites/scripts/scripts.js:498]: There are other projects that add the martech to the body, e.g.: https://github.com/hlxsites/elco-aerotop-sg/blob/main/scripts/third-party.js#L21-L24, https://github.com/hlxsites/mammotome/blob/main/scripts/scripts.js#L310-L315
The idea is not to encourage bloating the head with martech scripts, especially if at some point, the customer will have the idea to load them without delay, hence impacting the performance.

**Pull request merged**